### DISCUSSION TITLE
Current state of the mod

### DISCUSSION THREAD

**Discussion by Woprok** [Woprok/AOE4-AdvancedGameSettings#117] [Announcements]
I have decided to suspend the development and updating of all my mods indefinitely.
Full can be found:
https://www.reddit.com/r/aoe4/comments/17su85y/end_of_advanced_game_settings_mod/

**Comment by Woprok** [comment]
On that note, mod should work without any issue as long as you do not use DLC content (civs)

**Reply by Reak2382** [reply]
Ohh! so sad to hear but understandable.
Sadly this means I cannot use it to try out the DLC content but i wanted to tell you I am really happy and grateful for all the time you spent and for replying me. 
You brought my friends and myself much joy setting up cool treaty gameplay.

Thank you.

**Comment by RedEyeJhin** [comment]
can you fix or add the dlc faktions or are they hard coded ?
